tests/crypto/comp.lua

```diff
@@ -3,7 +3,11 @@
 -- SPDX-License-Identifier: MIT OR GPL-2.0-only
 --
 
-local comp = require"crypto.comp"
+local status, comp = pcall(require, "crypto.comp")
+if not status then
+	print("crypto.comp not available, skipping tests")
+	return
+end
```

Comment:
instead of doing this, we can register an empty library in this case.. we already do it on luaxdp if I'm not mistaken..